com.unity.ml-agents/Editor/DemonstrationDrawer.cs

```diff
@@ -23,9 +24,9 @@ internal class DemonstrationEditor : UnityEditor.Editor
         const string k_NumberStepsName = "numberSteps";
         const string k_NumberEpisodesName = "numberEpisodes";
         const string k_MeanRewardName = "meanReward";
-        const string k_ActionSpecName = "ActionSpec";
+        const string k_ActionSpecName = "m_ActionSpec";
```

Comment:
What was the problem here? Should we have a unit test that would have caught it?

(OK to do outside of this PR)

Reply:
I wrote this when we were adding an ActionSpec field, and later on we changed it to private field m_ActionSpec and public property ActionSpec. 

Since properties won't be serialized, `property.FindPropertyRelative("ActionSpec")` will get nothing and the demo import will fail. The exception is ignored though, resulted in demo files not imported properly and no information displayed in inspector, but no direct error message.

Reply:
Ah sorry, missed the fact that it was in the drawer.